spec/spec.md

```diff
@@ -656,7 +656,31 @@ If a message attempts to invoke an interface `method` that is not the implementa
 {
   "replies": [
     {
-      "status": { "code": 501, "detail": "The interface method is not implemented" }
+      "status": {
+        "code": 501,
+        "detail": "The interface method is not implemented"
+      }
+    }
+  ]
+}
+```
+:::
+
+**Resource consumption limit exceeded**
+
+If the DWeb Node instance receiving the request has determined that the rate of resource consumption has exceeded its tolerances and cannot process the request, the instance ****MUST**** respond with the following status entry:
```

Comment:
Related to [above](https://github.com/decentralized-identity/decentralized-web-node/pull/195/files?diff=split&w=1#r957646139), what is the point of `****` instead of `**`?